README.Rmd

```diff
@@ -26,75 +26,91 @@ knitr::opts_chunk$set(
+`install.packages()` and `devtools::install_github()`.
+pak is fast, safe and convenient.
+
+<p align="center">
+	<img width="1000" src="https://cdn.jsdelivr.net/gh/r-lib/pak@readme/tools/images/install-2.svg">
+</p>
+
 
 ## Installation
 
 Install the package from CRAN:
 
-``` r
+```r
 install.packages("pak")
 pak::pak_create_private_lib()
 ```
 
 The second line creates pak's own package library, to avoid interference
-between pak's dependencies and the user's regular packages. Run this
-command any time to update pak's package library.
+between pak's dependencies and the user's regular packages.
 
 ## Usage
 
-Simply call `pkg_install` to install packages:
+Simply call `pkg_install()` to install CRAN or BioConductor packages:
 
 ```r
-pak::pkg_install("dplyr", lib = "/tmp/lib")
+pak::pkg_install("usethis")
 ```
 
-All dependencies will be installed as well, to the same library.
-
-## Goals
+To install GitHub packages, use the `user/repo` syntax:
 
-* Make package installation cheaper and more reliable.
-
-* Cheaper, reliable package installation allows safer, more convenient
-  workflows.
+```r
+pak::pkg_install("r-lib/usethis")
+```
 
-## Features
+All dependencies will be installed as well, to the same library.
 
-### Speedy Features
+## Why pak?
 
-* Fast downloads and queries (all HTTP is concurrent).
+### Fast
 
-* Fast installs (package installations and builds are concurrent).
+* Fast downloads and HTTP queries. pak performs all HTTP requests
+  concurrently.
 
-* Package cache (all downloaded and locally built packages are cached).
+* Fast installs. pak builds and installs packages concurrently.
 
-* Lazy downloads, only download metadata and package files if needed.
+* Metadata and package cache. pak caches package metadata and all downloaded
+  packages locally. It does not download the same package files over and
+  over again.
 
-### Safety features
+* Lazy installation. pak only installs the packages that are really needed
+  to be installed. If the requested package and its dependencies are
+  already installed, pak does nothing.
 
-* Private library (pak's own dependencies do not affect your "regular"
-  packages and vice versa).
+### Safe
 
-* Do not load any package in the main process (except for pak itself).
-  Every operation runs in the sub-process, and the packages are loaded
-  from the private library.
+* Private library (pak's own package dependencies do not affect your
+  regular package libraries and vice versa).
 
-* Dependency solver: makes sure that you end up in a consistent, working
-  state of dependencies. Find conflicts up front, before actually installing
-  anything.
+* Dependency solver. pak makes sure that you end up in a consistent, working
+  state of dependencies. It finds conflicts up front, before actually
+  installing anything. pak fixes installations, i.e. if a downsteam
+  dependency is missing, it installs it, even if the requested package is
```

Comment:
I don't actually understand this example, could it be clarified or made more concrete?

Reply:
usethis (recursively) depends on mime. If you say `pkg_install("usethis")` and usethis is already installed, but mime is not, it will install mime only.

Reply:
But yeah, if you don't get it, I need to explain it better. Or remove it....

Reply:
Thanks, BTW!

Reply:
I understood it, but mostly because I watched the gif really carefully first.